frontend/views/containers/group-settings/GroupVotingSystem.vue

```diff
@@ -0,0 +1,151 @@
+<template lang='pug'>
+  ul.c-wrapper
+    li.cardBox(
+      v-for='type in votingSytemSorted'
+      :class='{ isActive: isVotingActive(type) }'
+      :aria-current='isVotingActive(type)'
+    )
+      p.is-title-4 {{ config[type].title }}
+      p.has-text-1.c-expl(v-html='config[type].explanation')
+      i18n.pill.is-primary.c-active(v-if='isVotingActive(type)') Active
+
+      dl.c-status(v-if='isVotingActive(type)')
+        dt.c-status-term {{ config[type].status }}
+        dd.c-status-desc
+          span(v-html='votingValue(type)')
+          i18n.link(
+            tag='button'
+            @click='openVotingProposal(type)'
+          ) Propose change
+
+      banner-simple.c-banner(
+        severity='info'
+        v-if='isVotingRuleAdjusted(type)'
+      )
+        | * &nbsp;
+        i18n This value was automatically adjusted because your group is currently smaller than the disagreement number.
```

Comment:
I don't know if other languages use a different symbol than `*` to indicate this. They might. They might also position it differently. So I think the `*` needs to be part of the `i18n`.